### DISCUSSION TITLE
We want to remove the color helpers from colors.css

### DISCUSSION THREAD

**Discussion by leonardorafael** [beercss/beercss#271] [Polls]
Hi,

We have colors in [theme.css](https://github.com/beercss/beercss/blob/main/src/cdn/helpers/theme.css) (used by theme) and [colors.css](https://github.com/beercss/beercss/blob/main/src/cdn/helpers/colors.css) (not used by theme).
I want to know if we can remove colors.css from default css.

Why we want to suggest the remove of colors?

1) The colors aren't in the MD3 (it's from MD1/MD2)
2) MD3 has the theme colors, so you can do the job just using the theme colors.
3) Without the colors, will reduce the final bundle from ~11.2kb to ~9.7kb (brotli).

**Comment by laurentpayot** [comment]
I voted no even if I use one (or two) colors from colors.css because I use purgecss to remove unused colors. I wouldn’t mind writing the colors in my custom CSS file.

**Comment by acroyear** [comment]
i'm a no - my colors are entirely set by material-dynamic-colors (though i may file some requests for improvements there - i need to isolate specific files (album covers) that don't produce the desired primary). even in my non-player aspects of my music app, i just set the colors using my app's logo.
